### PR TITLE
update dashboards, various JS fixes

### PR DESCRIPTION
**Pull request by ripienaar**
Signed-off-by: R.I.Pienaar <rip@devco.net>

### CODE REVIEW

**Comment by ripienaar** [comment]
@ColinSullivan1 initial dashboards had the data source incorrect so they wouldnt work, also a few fixes to make things work better with different accounts etc

**Comment by ripienaar** [comment]
@ColinSullivan1 think we can take a look at this one now

File: surveyor/surveyor_test.go [hunk 1]
```diff
@@ -214,31 +212,6 @@ func TestSurveyor_Reconnect(t *testing.T) {
 	}
 }
 
-func TestSurveyor_NoSystemAccount(t *testing.T) {
```
Comment (ripienaar) [inline surveyor/surveyor_test.go:217]: the changes in nats to always have a system account seem to have broken this test in weird ways so I just killed it for now

Comment (ColinSullivan1) [reply surveyor/surveyor_test.go:217]: OK.

**Review by ColinSullivan1 — APPROVED** [review]
LGTM.

**Pull request merged**